en/docs/install-and-setup/configuring-api-manager-for-ob.md

```diff
@@ -131,7 +131,130 @@ account retrieval. By default, this is disabled and the configuration is set to
+           ``` toml
+           [open_banking.uk.jws_signature.obie]
+           org_id="0015800001HQQrZAAX"
+        
+           [open_banking.uk.jws_signature.obie.trusted_anchors]
+           signature_validation="openbanking.org.uk"
+           ```
+
+         5. Configure the API contexts that require signature validation:
+          ``` toml
+          [[open_banking.uk.jws_signature.signature_validation.mandated_apis]]
+          api_context="/open-banking/v3.1/event"
+          ```
+
+13. To let the TPPs verify that the request wasn't tampered with, sign the responses:
+
+    !!! note
+        This is only available as a WSO2 Update from **WSO2 Open Banking API Manager UK Toolkit Level 3.0.0.x** and
+        **WSO2 Open Banking Identity Server UK Toolkit Level 3.0.0.x** onwards. For more information on updating, see
+        [Getting WSO2 Updates](setting-up-servers.md#getting-wso2-updates).
+
+    ??? tip "Click here to see JWS Response Signing configurations"
+
+         1. Configure the `UKJwsResponseHandlingExecutor` executor. The priority of `UKJwsResponseHandlingExecutor` 
+            must be lower than `ConsentEnforcementExecutor`. For example:
+           ``` toml 
+           [[open_banking.gateway.openbanking_gateway_executors.type.executors]]
+           name = "com.wso2.openbanking.accelerator.gateway.executor.impl.consent.ConsentEnforcementExecutor"
+           priority = 6
+
+           [[open_banking.gateway.openbanking_gateway_executors.type.executors]]
+           name = "com.wso2.openbanking.uk.gateway.executors.jws.UKJwsResponseHandlingExecutor"
+           priority = 999
+           ```
+        
+         2. Enable signing and define the response signing algorithms:
+           ``` toml
+           [open_banking.jws_signature.response_signing]
+           enabled=true
+           allowed_algorithm="PS256"
+           ```
+
+        3. Configure the alias and kid values of the signing certificates:
+            - `signing_cert_alias`: The alias of the signing certificate stored in the keystore. Used to sign responses in a production environment. Default value is `wso2carbon`.
+            - `sandbox_signing_cert_alias`: The alias of the signing certificate stored in the keystore. Used to sign responses in a sandbox environment. Default value is `wso2carbon`.
+            - `signing_cert_kid`: The kid value of the corresponding public key of the private key, which is used for signing in a production environment. Default value is `1234`. Mandatory configuration.
+            - `sandbox_signing_cert_kid`: The kid value of the corresponding public key of the private key, which is used for signing in a sandbox environment. Default value is `5678`. Mandatory configuration.
+           ```toml
+           [open_banking.ob_identity_retriever.server]
+           signing_cert_alias="wso2carbon"
+           sandbox_signing_cert_alias="wso2carbon"
+           signing_cert_kid="w7NFeMODzCMOZen_WECAlG9N8gg"
```

Comment:
shall we add 1234 and 5678 as default configs

Reply:
Fixed in 0913f39d3e68e6d5c0f106f6490a307c24c16758